packages/components/src/components/PasswordCreationField/components/FieldLabel/FieldLabel.module.scss

```
@@ -0,0 +1,22 @@
+.passwordFieldLabel {
```

Always use the file name as the "base" class name. Otherwise the resulting classname would be

```
flow--password-creation-field--field-label--password-field-label
```

instead of 

```
flow--password-creation-field--field-label
```

```suggestion
.fieldLabel {
```